frontend/src/components/Wheel/index.module.css

```diff
@@ -1,5 +1,127 @@
+.root {
+  text-align: center;
+}
+
 .align {
   display: grid;
   grid-template-columns: auto auto;
   width: 50%;
 }
+
+.circle {
+  position: relative;
+  width: 400px;
+  height: 400px;
+  padding: 0;
+  margin: 16px auto;
+  overflow: hidden;
+  list-style: none;
+  border: 1px solid black;
+  border-radius: 50%;
+  animation: 3s ease-in 1s 2 reverse both paused rotation;
+}
+
+.isRotating {
+  animation-play-state: running;
+}
+
+@keyframes rotation {
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.prize {
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 50%;
+  height: 50%;
+  overflow: hidden;
+  transform-origin: 0% 100%;
+}
+
+.text {
+  position: absolute;
+  left: -100%;
+  display: block;
+  width: 200%;
+  height: 200%;
+  padding-top: 20px;
+  font-size: 18px;
+  font-weight: bold;
+  text-align: center;
+  cursor: pointer;
+  transform: skewY(60deg) rotate(15deg);
+}
+
+.arrow {
+  z-index: 1;
+  display: inline-block;
+  margin-right: 10px;
+  margin-bottom: -33px;
+  border-top: 50px solid black;
+  border-right: 15px solid transparent;
+  border-left: 15px solid transparent;
+}
+
+.spinButton {
+  width: 120px;
+  height: 40px;
+  margin-top: 20px;
+  font-size: 18px;
+  color: white;
+  cursor: pointer;
+  background-color: blue;
+  border-radius: 30px;
+  outline: none;
+}
+
+@media only screen and (max-width: 350px) {
+  .wheelName {
+    margin-left: 65px !important;
+  }
+
+  .arrow {
+    margin-right: -60px !important;
+  }
```

Comment:
We should probably remove this `important` at same stage in the future

Reply:
Why are we merging PRs that aren't right al ready?